Makefile

```diff
@@ -1,6 +1,7 @@
 BUILD_DIR=$(shell pwd)/build
 BIN_DIR=$(BUILD_DIR)/bin
 HANDLE_USER_DATA=$(shell base64 -w 0 scripts/handle-user-data)
+YUM_WRAPPER=$(shell base64 -w 0 scripts/yum-wrapper)
```

Comment:
https://github.com/minishift/minishift-centos-iso/blob/master/Makefile#L40-L41 also need to be updated to pass it to kickstart file.